tests/test_runtime_sdk_integration.py

```
@@ -0,0 +1,1367 @@
+        assert fields["permission_mode"] == "bypassPermissions"
+        assert fields["system_prompt"]["append"].startswith("stable system prompt")
+        assert fields["tools"] == ["Agent"]
+        assert fields["mcp_servers"]["hermes-tools"]["tools"] == []
+        assert clients[0].queries == ["hello runtime"]
+        assert clients[0].disconnected == 1
+
+    asyncio.run(scenario())
+
+
+def test_runtime_preserves_safe_sdk_failure_code_and_retryability() -> None:
+    async def scenario():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `scenario`

Add return type annotation: `None`

(ANN202)